versioned_docs/version-24.04/releases/centreon-os.mdx

```diff
@@ -222,6 +222,20 @@ Many features have been added in this version.
 
 ## Centreon Collect
 
+### 24.04.6
+
+Release date: `September 25, 2024`
+
+<details>
+  <summary>Bug fixes</summary>
+
+- [Broker] when no filter is configured, forbidden filter is applied to all. 
+- [Engine] Fixed an issue regarding gRPC accepted services deletion.
+- [Engine] When the Engine configuration is modified, no need to send host/service status since their configuration are already sent.
+- [Engine] When Engine was restarted, its log file wasn't reopened, so if it was moved or removed, it wasn't created again.
```

Comment:
```suggestion
- [Broker] When no filter is configured on a Broker output, a default filter is applied so as to avoid Broker crashing.
- [Engine] Engine now waits until no thread uses a gRPC service before deleting the service.
- [Engine] When Engine's configuration is modified, already sent events are no longer sent a second time.
- [Engine] Fixed the following issue: when Engine was restarted, its log file wasn't reopened, so if it was moved or removed, it wasn't created again.
```